api/openapi.yaml

```diff
@@ -0,0 +1,393 @@
+openapi: 3.0.1
+info:
+  title: "Message Application"
+  description: >-
+    This application allows managing users, posts, comments, and followers.
+    Users may have posts that they have written, comments on other users' posts,
+    and followers which reference other users.
+  version: "1.0"
+
+servers:
+  - url: http://{host}:{port}
+    description: default
+    variables:
+      host:
+        default: localhost
+      port:
+        default: "5000"
+
+tags:
+- name: Users
+- name: Messages
+- name: Comments
+
+paths:
+  /users:
```

Comment:
I think we should have an endpoint along the lines of `/users/{id}/feed` that gets the messages from people who a user follows

Reply:
Sounds good, I'll add that in